specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -6467,6 +6476,9 @@
           "x-ms-client-flatten": true,
           "$ref": "#/definitions/WorkflowProperties",
           "description": "The workflow properties."
+        },
+        "identity": {
```

Comment:
I just want to double confirm. Does it mean the `identity` also needs to be set while creating workflow?

Reply:
it is an optional filed.. not a required. If you set it you get the identity.